src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private static final int BOUND = 100;
+
+    public Ball getRandomBall() {
+        Color color = new ColorSupplier().getRandomColor();
+        int number = new Random().nextInt(BOUND);
```

Comment:
rewrite the code so you don't unnecessarily recreate ColorSupplier and Random objects. Make them class fields

Reply:
I don't understand what actually i should do